Kernel/Interrupts/APIC.cpp

```diff
@@ -345,8 +345,9 @@ UNMAP_AFTER_INIT void APIC::setup_ap_boot_environment()
     // * aps_to_enable u32 values for ap_cpu_init_stacks
     // * aps_to_enable u32 values for ap_cpu_init_processor_info_array
     constexpr u64 apic_startup_region_base = 0x8000;
-    VERIFY(apic_startup_region_base + apic_ap_start_size < USER_RANGE_BASE);
-    auto apic_startup_region = create_identity_mapped_region(PhysicalAddress(apic_startup_region_base), Memory::page_round_up(apic_ap_start_size + (2 * aps_to_enable * sizeof(u32))).release_value_but_fixme_should_propagate_errors());
+    size_t apic_startup_region_size = apic_ap_start_size + (2 * aps_to_enable * sizeof(FlatPtr));
```

Comment:
Sorry for the nitpicking, but can we include the page_round_up here? since that's repeated as well.

Reply:
Well I didn't on purpose because the memcpy shouldn't copy more than the non-rounded value.

Reply:
But the memcpy does't use this variable

Reply:
Hm you're right. Shouldn't be doing PRs while distracted ;-)